### PR TITLE
Add posibility to configure the SVGO lib. from webpack options

### PR DESCRIPTION
**Pull request by boschr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "inline markups to HTML: SVG, MathML, etc.",
   "main": "index.js",
   "scripts": {
-    "test": "mocha ./test/index.spec.js"
+    "test": "node_modules/.bin/mocha ./test/index.spec.js"
```
Comment (asnowwolf) [inline package.json]: when we using `scripts`, the `node_modules/.bin` is default added to PATH, so we can use mocha only.

File: README.md [hunk 2]
```diff
@@ -6,23 +6,55 @@ This is a webpack loader. It can inline SVG or MathML file to HTML, so that you
...
 
-or with html-loader:
+Or with `html-loader`:
 
 ```js
 {
   test: /\.html$/,
-  loader: 'html!markup-inline'
+  use: [
+    'html',
```
Comment (asnowwolf) [inline README.md]: should change to `html-loader`?

File: README.md [hunk 3]
```diff
@@ -6,23 +6,55 @@ This is a webpack loader. It can inline SVG or MathML file to HTML, so that you
...
+},
+```
+
+Or with `html-loader` and a SVGO configuration. By default `markup-inline-loader` only enables the removeTitle plugin. You can overwrite this default behavior with following example:
+
+```js
+{
+  test: /\.html$/,
+  use: [
+    'html',
```
Comment (asnowwolf) [inline README.md]: should change to `html-loader`?

**Comment by asnowwolf** [comment]
Thank you for your help, I've add some comments.

**Comment by boschr** [comment]
You're right. I have processed your comments.

**Pull request merged**